### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Please review my cart page

### CODE REVIEW

**Comment by shruthipatil11** [comment]
Looks great.

File: pages/stylesheets/cart.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+.cards-horizontal{
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    border: 2px solid var(--wishlist);
+    position: relative;
+    max-width: 50rem;
+    border-radius: 5px;
+    /* min-height: 32rem; */
+    /* flex-wrap: wrap; */
```
Comment (dev-enforced) [inline pages/stylesheets/cart.css:38]: Can remove these comments

File: pages/stylesheets/products.css [hunk 2]
```diff
@@ -143,6 +142,8 @@ ul{
     background-color: var(--primary-color);
     color: var(--bg-code);
     font-size: 1.6rem;
+
+
```
Comment (dev-enforced) [inline pages/stylesheets/products.css:146]: Remove these extra spaces as well

**Review by dev-enforced — APPROVED** [review]
The code looks good to me!! Well structured code and use of variables and classes
And the site also looks good and the items are well spaced: https://deploy-preview-7--trekkart-store.netlify.app/pages/cart.html

**Pull request merged**